src/firebase/firebase.js

```diff
@@ -0,0 +1,60 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+
+const firebaseConfig = {
+    apiKey: "AIzaSyCXt-OPqpTDbbo2XPE_59B5hW-8A0_6Ry0",
```

Comment:
Shouldnt that come from a .env file in a variable? $firebaseApiKey for instance?

Reply:
I will check it, at the moment it is working.